src/CommunityVoices/App/Api/View/Image.php

```diff
@@ -109,12 +109,13 @@ protected function getImageNextImage()
 
         return $response;
     }
-    public function postImageUpdate()
+    protected function postImageUpdate()
     {
 
     }
-    public function postImageUnpair()
+    protected function postImageUnpair()
     {
-        
+        return $this->errorsResponse('imageUnpair');
```

Comment:
I checked and the image management service actually never acknowledges any errors.  I would probably just make it return `true` rather than an errors response because of this.

Reply:
done.